### PR TITLE
Optimize identify -verbose

### PR DESCRIPTION
**Pull request by kirkkwang**
The `identify -verbose` was too expensive of a method because it is getting back a ton of information that wasn't even being used.  We are opting for `identify -format` to pull only the information we are looking to use instead to help speed up the `CreateDerivativesJob` process.

The output looks something like:

```sh
Geometry: 5100x6592
Depth: 8
Colorspace: sRGB
Alpha: Undefined
MIME Type: JPEG
```

### CODE REVIEW

**Review by alishaevn — COMMENTED** [review]
looks good, but waiting for actual approval until the pipeline is green.

**Comment by jeremyf** [comment]
The below code is comparable to the post-commit state of https://github.com/scientist-softserv/iiif_print/pull/208

```shell
❯ rspec /Users/jeremy/git/space_stone-derivatives/spec/space_stone/derivatives/utilities/image_identify_spec.rb
...identify: CorruptImageProfile `/Users/jeremy/git/space_stone-derivatives/spec/fixtures/files/minimal-1-page.pdf' (XMP) @ warning/profile.c/ValidateXMPProfile/1935.
.

Finished in 0.25737 seconds (files took 0.36577 seconds to load)
4 examples, 0 failures
```

The below code is comparable to the pre-commit state of https://github.com/scientist-softserv/iiif_print/pull/208

```shell
❯ rspec /Users/jeremy/git/space_stone-derivatives/spec/space_stone/derivatives/utilities/image_identify_spec.rb
...identify: CorruptImageProfile `/Users/jeremy/git/space_stone-derivatives/spec/fixtures/files/minimal-1-page.pdf' (XMP) @ warning/profile.c/ValidateXMPProfile/1935.
.

Finished in 4.99 seconds (files took 0.27182 seconds to load)
4 examples, 0 failures
```

**Comment by ShanaLMoore** [comment]
> looks good, but waiting for actual approval until the pipeline is green.

Thanks for checking it out @alishaevn looks like we've got 2 specs to fix! Please standby.

**Review by laritakr — APPROVED** [review]

**Pull request merged**
